### PR TITLE
*: support admin split region.

### PR DESCRIPTION
**Pull request by disksing**

### CODE REVIEW

File: pdctl/command/operator.go [hunk 1]
```diff
@@ -227,6 +228,34 @@ func removePeerCommandFunc(cmd *cobra.Command, args []string) {
 	postJSON(cmd, operatorsPrefix, input)
 }
 
+// NewSplitRegionCommand returns a command to split a region.
+func NewSplitRegionCommand() *cobra.Command {
+	c := &cobra.Command{
+		Use:   "remove-peer <region_id>",
```
Comment (Connor1996) [inline pdctl/command/operator.go]: remove-peer ?

File: server/handler.go [hunk 2]
```diff
@@ -398,6 +398,24 @@ func (h *Handler) AddRemovePeerOperator(regionID uint64, fromStoreID uint64) err
...
+	}
+
+	region := c.cluster.GetRegion(regionID)
+	if region == nil {
+		return errRegionNotFound(regionID)
+	}
+
+	step := schedule.SplitRegion{StartKey: region.StartKey, EndKey: region.EndKey}
+	op := schedule.NewOperator("adminSplitRegion", regionID, schedule.OpAdmin, step)
+	c.addOperator(op)
```
Comment (Connor1996) [inline server/handler.go]: do we need to return whether add operator successfully?

File: pdctl/command/operator.go (hunk 1 above)
Comment (disksing) [reply pdctl/command/operator.go]: :cry: typo.

File: server/schedule/operator.go [hunk 3]
```diff
@@ -104,6 +105,31 @@ func (rp RemovePeer) Influence(opInfluence OpInfluence, region *core.RegionInfo)
...
+	StartKey, EndKey []byte
+}
+
+func (sr SplitRegion) String() string {
+	return "split region"
+}
+
+// IsFinish checks if current step is finished.
+func (sr SplitRegion) IsFinish(region *core.RegionInfo) bool {
+	return !bytes.Equal(region.StartKey, sr.StartKey) || !bytes.Equal(region.EndKey, sr.EndKey)
```
Comment (siddontang) [inline server/schedule/operator.go]: how about we split but merge again, so here the check may still return false?

Comment (disksing) [reply server/schedule/operator.go]: Merge checker will skip this region if it already has an operator.

**Pull request merged**